docs/en/tutorials/getting-started.rst

```diff
@@ -186,7 +194,10 @@ doctrine command. Its a fairly simple file:
     // cli-config.php
     require_once "bootstrap.php";
     
-    return \Doctrine\ORM\Tools\Console\ConsoleRunner::createHelperSet($entityManager);
+    $helperSet = ConsoleRunner::createHelperSet($entityManager->getConnection());
+    // register the entity manager helper
+    $helperSet->set(new EntityManagerHelper($entityManager), 'em');
```

Comment:
If we are miss the `'em'` helper, then we should get rid of it from the docs instead (if the name doesn't match)